src/app/app.component.ts

```diff
@@ -178,4 +203,19 @@ export class AppComponent implements OnDestroy, OnInit, AfterViewInit {
     NotificationCenterComponent.outlineSelectedNotification(notification.id);
   }
 
+
+  getBackground(level) {
+    const color = this.adminMessageLevels.find((l) => l.value == level).color;
+    return `bg-${color}-100`
```

Comment:
i dont like this approach since it's in the HTML page and uses a find to collect the color.

Better optimized would be a switch for the levels that return the color or (if we want to work with the const) change it to a dict since the a level wont exist twice